### DISCUSSION TITLE
How to install karee

### DISCUSSION THREAD

**Discussion by Guiffou-Joel** [ChamplainLeCode/karee-cli#3] [Q&A] [answered]
Salut a tous!
J'aimerais installer karee sur ma machine mais je ne trouve pas la procedure d'installation.
Quelqu'un peut m'aider?

**Comment by ChamplainLeCode** [comment, marked as answer]
Salut @Guiffou-Joel, La documentation est fait sur [Wiki](https://github.com/ChamplainLeCode/karee/wiki/Karee-CLI---installation). Rends-toi y et tu trouveras la procédure et la suite.

néanmoins tu peux exécuter
```bash
    npm install -g karee
```

**Comment by Guiffou-Joel** [comment]
Daccord merci
